.circleci/config.yml

```diff
@@ -577,13 +629,6 @@ jobs:
       - run:
           name: forge version
           command: forge --version
-      - run:
-          name: solc warnings check
-          command: |
-            forge build --force --deny-warnings || echo "export SOLC_WARNINGS_CHECK=1" >> "$BASH_ENV"
-          environment:
-            FOUNDRY_PROFILE: ci
-          working_directory: packages/contracts-bedrock
```

Comment:
No more warnings check?

Reply:
Warnings check has moved into contracts-bedrock-build instead to save time